lib/models/api_calls.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:convert';
+import 'dart:io';
+import 'package:http/http.dart' as http;
+import 'package:chrome_management_app/objects/account_devices.dart';
+import 'package:chrome_management_app/objects/detailed_device.dart';
+
+/// The model will manage all the API calls to the Directory API.
+///
+/// See more
+/// https://developers.google.com/admin-sdk/directory/v1/guides/manage-chrome-devices.
+
+class ApiCalls {
+  ///Url to get the intial page of the list of devices
+  static final _deviceListUrl =
```

Comment:
For class constant, we use DEVICE_LIST_URL

Reply:
Done.